Pages/js/options.js

```diff
@@ -152,25 +156,27 @@ function saveWhitelist() {
  * Restores select box state to saved value from local storage.
  */
 function restoreWhitelist() {
-  
-  restoreServer();
-  
+
```

Comment:
@smcgregor Yes, I check for CSV during the restore and convert it to the new JSON format.